### PR TITLE
Create mypython.py

### PR DESCRIPTION
**Pull request by ashika1990**
this is to add tanisha and ashika with some space

### CODE REVIEW

**Comment by arafatheds** [comment]
Please go ahead and merge

File: mypython.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+first_name=('tanisha')
+second_name=('ashika')
+names=first_name+ '' +second_name
```
Comment (arafatheds) [inline mypython.py]: Please remove the single quotes in the line 3 and let me know

Comment (ashika1990) [reply mypython.py]: i removed single quote.pklease check

Comment (arafatheds) [reply mypython.py]: Good.. I will approve now

**Review by arafatheds — APPROVED** [review]

**Pull request merged**